plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/AbstractTestParquetReader.java

```diff
@@ -884,17 +886,21 @@ public void testDecimalBackedByINT32()
                     intValues.stream()
                             .map(value -> SqlDecimal.of(value, expectedPrecision, scale))
                             .collect(Collectors.toList()),
+                    getOnlyElement(TEST_COLUMN),
```

Comment:
Sounds like TEST_COLUMN shouldn't be a list

Reply:
It's a mixed bag, if we try to make it a String then certain places which expect a List need to be changed, so I'll leave this unchanged for now.